chapters/en/chapter10/2.mdx

```diff
@@ -0,0 +1,63 @@
+# Set up your Argilla instance[[set-up-your-argilla-instance]]
+
+<CourseFloatingBanner chapter={10}
+  classNames="absolute z-10 right-0 top-0"
+  notebooks={[
+    {label: "Google Colab", value: "https://colab.research.google.com/github/huggingface/notebooks/blob/master/course/en/chapter10/section2.ipynb"},
+    {label: "Aws Studio", value: "https://studiolab.sagemaker.aws/import/github/huggingface/notebooks/blob/master/course/en/chapter10/section2.ipynb"},
+]} />
+
+To start using Argilla, you will need to set up your own Argilla instance first. Then you will need to install the Python SDK so that you can manage Argilla using Python code.
+
+## Deploy the Argilla UI
+
+The easiest way to set up your Argilla instance is through Hugging Face Spaces:
+
+1. Go to [Spaces](https://huggingface.co/spaces) in the Hugging Face Hub, and then click **Create new Space**.
+2. Under "Select the Space SDK" select **Docker**, then **Argilla**.
+3. Fill in the rest of the form following the instructions and click **Create Space**.
+
+<img src="https://huggingface.co/datasets/huggingface-course/documentation-images/resolve/main/en/chapter10/space_config.png" alt="Space configuration form."/>
+
+>[!WARNING]
+> ⚠️ You may want to enable **Persistent storage** so the data isn't lost if the Space is paused or restarted.
+> You can do that from the Settings of your Space.
+
+>[!TIP]
+> 💡 Alternatively, you may want to deploy Argilla locally instead of using a Hugging Face Space. You can do this using a Docker image as explained in [this guide](https://docs.argilla.io/dev/getting_started/how-to-deploy-argilla-with-docker/).
+
+Once Argilla is up and running, you can log in with your credentials.
+
+## Install and connect the Python SDK
```

Comment:
Maybe you've thought about this already and discarded this option, so feel free to ignore it. I think showing how to do it with import from UI would mean more users finishing the chapter and a nicer onboarding.